istio/release/samples/bookinfo/kube/route-rule-http-redirect.yaml

```diff
@@ -1,7 +1,8 @@
-apiVersion: config.istio.io/v1alpha3
```

Comment:
is this old rule? I dont' see in Istio release. if so please delete this

Reply:
as agreed to keep it.

Reply:
it should be updated, does not work otherwise.